blog/_posts/2018-07-13-team-akaringular.md

```diff
@@ -0,0 +1,36 @@
+---
+**title:** New Tech Journey With RGSoC!
+**layout:** post
+**created_at:** Fri Jul 13 2018
+**permalink:** blog/2018-07-13-team-akaringular
+**current:** blog
+**author:** Rachael Kiteme
+**twitter:** rachaelkiteme
+**categories:**  news
```

Comment:
This should be under 'student-posts' category. Please check the format [from other files] to declare categories parameter as well.